targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Public/PlayFabCommonSettings.h.ejs

```diff
@@ -38,7 +38,7 @@ namespace PlayFabCommon
         {
             if (serverURL.Len() == 0)
             {
-                serverURL = TEXT("https://") + (!verticalName.IsEmpty() ? verticalName : titleId) + productionEnvironmentURL;
+                serverURL = TEXT("https://") + titleId + (verticalName.IsEmpty() ? "" : "." + verticalName) + productionEnvironmentURL;
```

Comment:
This isn't correct.
Revert this line, because the correct fix is more complex, and if it's not the correct change, then we shouldn't bother with it.

Reply:
Can you please explain why this isn't correct?
Based on my understanding, the URL construction we have is title.vertical.playfabapi.com.
With the current URL construction achieving title.vertical is pretty much impossible and so it results in us not being able to use this code for testing the feature on our own private verticals.

Reply:
Furthermore, I'd like to remind you about the fact that a similar change was checked into the unity-v2 target a little earlier as part of our [work for local-debugging in Unity](https://github.com/PlayFab/SDKGenerator/commit/7ed2127df3ccf74a7a7441c57baa43587a2dcc30).

Reply:
Let's talk in person about the url construction.
In short: it's way more complicated.

Reply:
Resolving as per our email thread conversation regarding keeping this code the way it is and renaming `vertical` to `cloud` in a subsequent PR either by our team or yours.